src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__rate-count {
+    font-size: 10px;
+    line-height: 14px;
+    align-self: flex-end;
+    columns: $main-color;
+  }
+
+  &__price {
+    display: flex;
+    margin-bottom: 16px;
+    justify-content: space-between;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-num {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    text-align: center;
+    line-height: 40px;
+    color: #fff;
+
+    background-color: $blue-color;
+    border-radius: 5px;
+  }
+
+  &__button:hover {
+    color: $blue-color;
+    background-color: #fff;
+    border: 1px solid $blue-color;
```

Comment:
the border must be not only on the hover button but always, and you should change its color